corehq/sql_accessors/migrations/0056_add_hashlib_functions.py

```diff
@@ -13,8 +14,12 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        # this originally installed the hashlib extension in production as well
+        # but commcare-cloud does that where possible already
+        # and Amazon RDS doesn't allow it
         HqRunSQL(
             'CREATE EXTENSION IF NOT EXISTS hashlib',
             'DROP EXTENSION hashlib'
-        ),
+        )
+        if settings.UNIT_TESTING else noop_migration()
```

Comment:
This feels wrong. Can you make this a no-op and if it's needed for tests then add it to the test setup in .travis.yml or something?

Reply:
Or actually if possible it'd be nice to have it in the docker file

Reply:
You mean by running `psql -c CREATE EXTENSION IF NOT EXISTS hashlib DROP EXTENSION hashlib` directly? I'm open to that. Can you articulate what feels wrong about doing it here (and better about doing it in test setup?)

Reply:
> You mean by running psql -c CREATE EXTENSION IF NOT EXISTS hashlib directly

To be clear I mean that this migration is only `noop_migration()` and removing anything being run here. The extension would be created either in:

A. the docker setup https://github.com/dimagi/docker-postgresql
B. Or in the script that runs tests (after docker is setup, but before any test code is run)
 
> Can you articulate what feels wrong about doing it here

I think there are a few things that I don't like about this.

A. I'd like to keep `settings.UNIT_TESTING` to a minimum in production code
B. 
> this originally installed the hashlib extension in production as well but commcare-cloud does that where possible already and Amazon RDS doesn't allow it

Sounds like we have independently decided that this kind of database maintenance belongs in database setup and not sprinkled in the code so it makes sense to remove these.

C. If I want to figure out what differences between my local setup vs testing vs production it adds one more place I need to look and forget about.

As an aside I think what I'd really like is to not have the extension installed on any production instances at all if it can't be installed on all of them as I could see that leading to some hard to figure out issues for the devops team.